docs/en/introduction/comparison/juicefs_vs_alluxio.md

```diff
@@ -29,48 +30,48 @@ The following table shows the differences of main features between Alluxio and J
 
 ## Storage format
```

Comment:
this section could use more elaboration: sure, they have different storage format, but what does it really mean for end-users? talk more on performance differences on different scenarios cc @xiaogaozi